pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, openblas
+, libcint
+, libxc
+, h5py
+, numpy
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pyscf";
+  version = "1.7.0";
+
+  # must download from GitHub to get the Cmake & C source files
+  src = fetchFromGitHub {
+    owner = pname;
```

Comment:
This is a case of *incidental* equality. While `owner` and `repo` happen to be the same as `pname`, they do not *change* together. The  name might change because of a rename while staying in the same repo. Or we might decide to switch to a fork.

Therefore it doesn't make any sense to use `pname` for `owner` and `repo`. It doesn't hurt much either, but it is a bit of a pet-peeve of mine.